### ISSUE TITLE
Problems with connecting

### ISSUE THREAD

**Issue by ninokotur** [croneter/PlexKodiConnect#304]
Hi guys,

I have issues with PlexKodiConnect.

I configured it by the guide in PKC readme, however its throwing issue.
`20:15:29 T:12196  NOTICE: PLEX.userclient -> DEBUG: Returning active server: http://192.168.2.93:32400
20:15:29 T:12196  NOTICE: PLEX.userclient -> DEBUG: Server found: http://192.168.2.93:32400
20:15:29 T:12196  NOTICE: PLEX.userclient -> DEBUG: Authenticating user
20:15:29 T:12196  NOTICE: PLEX.userclient -> DEBUG: Returning active server: http://192.168.2.93:32400
20:15:29 T:12196  NOTICE: PLEX.userclient -> DEBUG: Trying to authenticate with old settings
20:15:29 T:12196  NOTICE: PLEX.userclient -> DEBUG: Loading current user
20:15:29 T:12196  NOTICE: PLEX.userclient -> DEBUG: Returning active server: http://192.168.2.93:32400
20:15:29 T:12196  NOTICE: PLEX.userclient -> DEBUG: Testing validity of current token
20:15:29 T:12196  NOTICE: PLEX.PlexAPI -> DEBUG: Checking connection to server http://192.168.2.93:32400/library/onDeck with verifySSL=False
20:15:39 T:12196  NOTICE: PLEX.downloadutils -> DEBUG: Server timeout at: http://192.168.2.93:32400/library/onDeck
20:15:39 T:12196  NOTICE: PLEX.downloadutils -> DEBUG: HTTPConnectionPool(host='192.168.2.93', port=32400): Read timed out. (read timeout=10)
20:15:39 T:12196  NOTICE: PLEX.PlexAPI -> DEBUG: Could not connect to http://192.168.2.93:32400/library/onDeck
20:15:40 T:12196  NOTICE: PLEX.PlexAPI -> DEBUG: Failed to connect to http://192.168.2.93:32400/library/onDeck too many times. PMS is dead`

I recently installed this plex server on debian 8. I have disabled all firewalls of all computers included. Plex through web and on every other device is working properly, I'm just not able to make it work through Kodi. I've tried with SSL on and off and also with manual config of server inside addon settings 

What i found interesting is when i go to link http://192.168.2.93:32400/library/onDeck thats reporting issue in log. I get 401 unauthorized (i assume that this could be related to API keys? not sure)

Can anyone please help me out with this issue.

Apologies if I missed form or something this is my first post on github. o/

Thank you.

**Comment by croneter** [comment]
Can you manually enter your PMS IP to connect?

**Comment by ninokotur** [comment]
Hi, 

This is manual settings, I'm not getting any gui error with it however log is getting flooded. On auto settings I'm getting gui error problems with connecting to server

**Comment by ninokotur** [comment]
I have tried resetting addon and kodi settings. I'm just confused about that 401 upon visiting lino I've mentioned. Still not sure if that is api key related or not. Because from this point of view it seems legitimate that it's unable to connect if I'm getting 401 on link that I can't even manually open... 

Any suggestions?

**Comment by croneter** [comment]
Please share the entire log with PKC debug logging enabled

**Comment by ninokotur** [comment]
Is it late for you in 1 hour or so? I'm just getting to work but I have openvpn with home so there won't be issues with that

**Comment by ninokotur** [comment]
Hi,

Here is entire log that i have. I hope you find this helpful.

https://pastebin.com/ifcr39Mq

Please let me know if there is a way i can assist you further.



Thanks.

**Comment by ninokotur** [comment]
Hello,

Is there any update regarding this issue?


Thanks.

**Comment by ninokotur** [comment]
croneter have you been able to identify issue?

**Comment by croneter** [comment]
Yes. It needs complete rewiring of the connection manager, so lots of works

**Comment by croneter** [comment]
Root cause is that your PMS does not answer within 10 seconds, so PKC assumes it's dead

**Comment by ninokotur** [comment]
Oh, that doesn't sound good... 

Is there something I can do to get it up temporarily?

**Comment by croneter** [comment]
@ninokotur 
Could you try the new BETA version 2.0.30 and see if it helps?
